### PR TITLE
feat: Update combine script and add final files

### PR DESCRIPTION
**Pull request by cc7768**
Update combine script to generate MR json

Also add final output files

### CODE REVIEW

File: combine_rewards.py [hunk 1]
```diff
@@ -1,4 +1,10 @@
+import decimal
+import json
+
+from decimal import Decimal
```
Comment (pxrl) [inline combine_rewards.py:4]: Importing decimal.Decimal and Decimal concurrently?

File: combine_rewards.py [hunk 2]
```diff
@@ -22,14 +28,69 @@
...
+    runningSum += totalDecimal*zeros
+    out.at[address, "total"] = totalDecimal
+
+    entry = {
+        "account": address,
+        "amount": str(removeDecimals(totalDecimal*zeros)),
+        "metadata": {
+            "amountBreakdown": {
+                "communityRewards":  str(removeDecimals(row["community"] * zeros)),
+                "welcomeTravelerRewards": str(removeDecimals(row["bridge-traveler"] * zeros)),
```
Comment (pxrl) [inline combine_rewards.py:82]: Assume welcomeTraveler is needed for legacy reasons?

**Review by pxrl — APPROVED** [review]
Looks OK!

File: combine_rewards.py (hunk 1 above)
Comment (cc7768) [reply combine_rewards.py:4]: I was originally using `decimal.?` for something else but it got nuked -- Removed this.

File: combine_rewards.py (hunk 2 above)
Comment (cc7768) [reply combine_rewards.py:82]: Exactly -- This was the output I was told to replicate.

**Pull request merged**